### ISSUE TITLE
Accessibility issue with "<div class="ss-list" role="listbox">"

### ISSUE THREAD

**Issue by bagaga** [brianvoe/slim-select#339]
**Describe the bug**
I use "slim-select" version 1.27.0. and I have an accessibility issue. I did the scan of my website with [lighthouse plugin](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?hl=fr), and as report, it gives me that the `<div class="ss-list" role="listbox">` does not have accessible name
**Screenshots**
![image](https://user-images.githubusercontent.com/11464786/169015181-1e379a4b-a915-49aa-934b-af1cf794ffc2.png)

I found in "slim-select" version 1.27.1 there is a commented code https://github.com/brianvoe/slim-select/blob/v1.27.1/src/slim-select/slim.ts#L558 , but I dont know if the resolution will be in future releases.

Please let me know if it will be in future releases.

**Comment by brianvoe** [comment]
This feature seems to be in there now. New release coming soon.
